balloon/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "balloon-hash"
+version = "0.1.0" # Also update html_root_url in lib.rs when bumping this
+description = "Pure Rust implementation of the Balloon password hashing function"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+documentation = "https://docs.rs/balloon-hash"
+repository = "https://github.com/RustCrypto/password-hashes/tree/master/balloon"
+keywords = ["crypto", "password", "hashing"]
+categories = ["cryptography", "no-std"]
+edition = "2018"
+readme = "README.md"
+
+[dependencies]
+digest = { version = "0.9", default-features = false }
+crypto-bigint = { version = "0.2", git = "https://github.com/RustCrypto/crypto-bigint", features = ["generic-array"] }
```

Comment:
This needs an update now that https://github.com/RustCrypto/crypto-bigint/pull/14 is merged.
https://github.com/RustCrypto/crypto-bigint/pull/17 will allow us to use `default-features = false` here.

Reply:
`default-features = false` was re-introduced.

Reply:
All relevant PRs were merged, so this is only waiting for the next release.

Reply:
Release PR here https://github.com/RustCrypto/crypto-bigint/pull/23

Reply:
Thanks, done!

Reply:
Done.